azure/packages/test/end-to-end-tests/package.json

```diff
@@ -38,12 +39,12 @@
 		"cache-dir": "nyc/.cache",
 		"exclude": [
 			"src/test/**/*.*ts",
```

Comment:
pre-existing, but this coverage setup config seems odd if not incorrect. The only files in this package are test files. No need to change, but I'd be curious to understand why we bother with coverage of nothing... or if that is not the effect.?

Reply:
[AB#7424](https://dev.azure.com/fluidframework/internal/_workitems/edit/7424)